Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'bootsnap', '>= 1.4.4', require: false
 gem 'rack-cors', require: 'rack/cors'
 gem 'active_model_serializers'
 gem 'devise_token_auth', github: 'lynndylanhurley/devise_token_auth', branch: 'master'
+gem 'faker', git: 'https://github.com/faker-ruby/faker.git', branch: 'master'
```

Comment:
can you explain what this does?

Reply:
This can generate random data. We use it to generate an email for the factory, so that every article in the test file will have unique emails